forge-gui/res/cardsfolder/upcoming/sanctuary_warden.txt

```diff
@@ -0,0 +1,16 @@
+Name:Sanctuary Warden
+ManaCost:4 W W
+Types:Creature Angel Soldier
+PT:5/5
+K:Flying
+K:etbCounter:Shield:2
+

```

Comment:
Empty lines would need to be removed from the final script.

Reply:
Done.